customization/dark-theme.md

```diff
@@ -0,0 +1,212 @@
+
+  2. Change Style to `Albatross`
+
+**Note:** The black appearance theme `Xfce-dusk` makes the VM names in the `Qubes OS Manager` unreadable.
+
+2 - *(Optional)* Change Window Manager Style
+
+  1. Open the `Window Manager` dialog
+
+    ~~~
+    Qubes Menu -> System Tools -> Appearance
+    ~~~
+
+    ![window manager dialog](/attachment/wiki/Dark-Theme/xfce-window-manager-theme.png)
+
+  2. Change the Theme in the `Style` Tab (e. g. Defcon-IV). All available themes work.
+
+
+Dark App VM, Template VM, Standalone VM, HVM (Linux Gnome)
+==========================================================
+
+Almost all Qubes VM's are based on the Gnome desktop. Therefor the description below is focused on the Gnome Desktop Environment.
+
+Using `Gnome-Tweak-Tool`
+------------------------
+
+The advantage of creating a dark themed Template VM is, that each AppVM which is derived from the Template VM will be dark themed by default.
+
+**Note:** Gnome-Tweak-Tool crashes under Archlinux. A workaround is to assign the AppVM to another TemplateVM (Debian, Fedora) which has Gnome-Tweak-Tool installed. Start the AppVM and configure the settings. Shutdown the machine and switch the template VM back to Archlinux.
+
+0 - Start VM
+
+**Note:** In case of App VM start the Template on which the AppVM is based on.
+
+1 - Install `Gnome-Tweak-Tool`
+
+  1. Fedora
+
+    ~~~
+    sudo dnf install gnome-tweak-tool
+    ~~~
+
+  2. Debian
+
+   ~~~
+   sudo apt-get install gnome-tweak-tool
+   ~~~
+
+2 - *(Only AppVM)* Stop template and start AppVM
+
+3 - Add `Gnome-Tweak-Tool` to the Application Menu
+
+  1. `Right-click` on VM entry in `Qubes VM Manager` select `Add/remove app shortcuts`
+
+  2. Select `Tweak Tool` and press the `>` button to add it
+
+  ![Application Dialog](/attachment/wiki/Dark-Theme/dialog-add-gnome-tweak-tool.png)
+
+4 - Enable `Global Dark Theme`
+
+ 1. *Debian only*
+
+  ~~~
+  cd ~/.config/
+  mkdir gtk-3.0
+  cd gtk-3.0/
+  touch settings.ini
+  ~~~
+
+ 2. Start `Tweak Tool` from the VM application menu and set the `Global Dark Theme` switch to `on`
+
+ ![Global Dark Theme enabled](/attachment/wiki/Dark-Theme/gnome-tweak-tool.png)
+
+5 - *(Optional)* Modify Firefox
+
+**Note:** Firefox uses GTK style settings by default. This can create side effects such as unusable forms or search fields. There are two different ways to avoid this. Either by using a add-on or by overwriting the defaults.
+
+  - use the theme [GTK+ Dark Theme Global Fixes](https://userstyles.org/styles/111694/gtk-dark-theme-global-fixes) and the [Stylish](https://addons.mozilla.org/en-US/firefox/addon/stylish/) addon
+
+  - or add the following line to `/rw/config/rc.local`
+
+    ~~~
+    sed -i.bak "s/Exec=firefox %u/Exec=bash -c 'GTK_THEME=Adwaita:light firefox %u'/g" /usr/share/applications/firefox.desktop
+    ~~~
+
+6 - Restart VM or all application
+
+Manually
+--------
+
+Manually works for Debian, Fedora and Archlinux.
+
+0 - Start VM
```

Comment:
Please don't use this format for making a numbered list. Instead, always use Markdown format. If you need more levels of organization, use headings.

In other words, instead of this:

1 - Foo
3 - Foo
3 - Foo

Do this:

1. Foo
2. Foo
3. Foo